docs/user-guides/workspace-access/jetbrains/jetbrains-toolbox.md

```diff
@@ -0,0 +1,65 @@
+# JetBrains Toolbox Integration
+
+JetBrains Toolbox helps you manage JetBrains products and includes remote development capabilities for connecting to Coder workspaces.
+
+## Before you begin
+
+- Install [JetBrains Toolbox](https://www.jetbrains.com/toolbox-app/) version 2.6.0.40284 or later
+- Ensure your Coder workspace [has the necessary IDE backends installed](../../../admin/templates/extending-templates/jetbrains-gateway.md)
+
+## Install the Coder plugin for Toolbox
+
+1. Open Toolbox and navigate to the **Remote Development** section.
+1. Install the Coder plugin using one of these methods:
+   - Search for `Coder` in the **Remote Development** plugins section.
+   - Use this URI to install directly: `jetbrains://gateway/com.coder.toolbox`.
+   - Download from [JetBrains Marketplace](https://plugins.jetbrains.com/).
```

Comment:
Can we add screenshots here? I think the best flow would be to show the screenshot and remove all these sub points.